projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -357,10 +335,133 @@ function enqueue_assets() {
 			 */
 			do_action( 'jp_carousel_enqueue_assets', $this->first_run, $localize_strings );
 
+			// Add the carousel skeleton to the page.
+			$this->localize_strings = $localize_strings;
```

Comment:
The reason I kept this as-is is because `$localize_strings` is populated by other wpcom-specific mu-plugins, as well as provided as a parameter to at least one action. I feared removing some of the values that are now no longer directly used in JS could introduce compatibility issues.

That said, I could still create `$this->skeleton_strings` as a subset of `$localize_strings` with just the values that are needed for the skeleton, but I'm not sure how useful that would be.